util/types/schemas/ckb.mol

```diff
@@ -155,38 +155,38 @@ struct TransactionKey {
 }
 
 table TransactionInfo {
```

Comment:
Make the `TransactionInfo` as struct?

Reply:
It's not an on-chain data structure, which is beyond the scope of this PR.